cxl/lib/libcxl.c

```diff
@@ -13955,3 +13955,72 @@ CXL_EXPORT int cxl_memdev_core_volt_get(struct cxl_memdev *memdev)
 	cxl_cmd_unref(cmd);
 	return rc;
 }
+
+#define CXL_MEM_COMMAND_ID_OEM_ERR_INJ_VIRAL CXL_MEM_COMMAND_ID_RAW
+#define CXL_MEM_COMMAND_ID_OEM_ERR_INJ_VIRAL_OPCODE 64289
```

Comment:
Any specific reason to use opcode as a decimal value as instead of hex.
Better to use hex to be consistent with other opcodes

Reply:
Not sure why many of the opcodes are in decimal, will change these to hex to be in consistent and for better readability.